js/client.js

```diff
@@ -2475,10 +2478,60 @@
 			this.close();
 			app.focusRoom('');
 			app.rooms[''].focusPM(this.data.name);
+		},
+		toggleOptions: function () {
+			this.moreOptions = !this.moreOptions;
+			if (this.moreOptions) {
```

Comment:
This is redundant. `app.addPopup` will already close a popup if it's already open (see how the top-right gear menu is implemented). You don't need to keep track of `this.moreOptions` at all.